src/gcp/pci-dss-3.2.1/rules/pci-dss-3.2.1-1.1.ts

```diff
@@ -0,0 +1,127 @@
+//GCP CIS 1.2.0 Rule equivalent 4.3
+export default {
+  id: 'gcp-pci-dss-3.2.1-1.1',
+  title: 'Compute instance "block-project-ssh-keys" should be enabled',
```

Comment:
To be consistent with the others PCI benchmarks, for example [AWS PCI](https://github.com/cloudgraphdev/cloudgraph-policy-packs/tree/main/src/aws/pci-dss-3.2.1/rules), you should use the same naming convention for the file name, id and title of the rules, you can also see in this PR for [GCP PCI](https://github.com/cloudgraphdev/cloudgraph-policy-packs/pull/40/files#diff-e42a7e5c48c8193ede4fe509d6d0ab1917fad3387720496996fe2ecbcbd533bd).
These are grouped by category, for example for your rule 1.1 (Virtual Machine) you could name it like this:

```
id: 'gcp-pci-dss-3.2.1-vm-check-1'
title: 'VM check 1: Compute instance "block-project-ssh-keys" should be enabled'
```